apps/back/migrations/Version20230906111240.php

```diff
@@ -7,26 +7,21 @@
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\Migrations\AbstractMigration;
 
-/**
- * Auto-generated Migration: Please modify to your needs!
- */
-final class Version20230905083639 extends AbstractMigration
+final class Version20230906111240 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return '';
+        return 'Database main for knp-taste application';
     }
 
     public function up(Schema $schema): void
     {
-        // this up() migration is auto-generated, please modify it to your needs
         $this->addSql('CREATE TABLE courses (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(180) NOT NULL, video VARCHAR(180) NOT NULL, UNIQUE INDEX UNIQ_A9A55A4C5E237E06 (name), UNIQUE INDEX UNIQ_A9A55A4C7CC7DA2C (video), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
-        $this->addSql('CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE user (id INT AUTO_INCREMENT NOT NULL, email VARCHAR(180) NOT NULL, username VARCHAR(180) NOT NULL, roles JSON NOT NULL, password VARCHAR(255) NOT NULL, is_verified TINYINT(1) NOT NULL, register_at DATETIME NOT NULL, video_viewed INT NOT NULL, UNIQUE INDEX UNIQ_8D93D649E7927C74 (email), UNIQUE INDEX UNIQ_8D93D649F85E0677 (username), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```

Comment:
Perhaps it is better to use uuid instead of id.
https://symfony.com/doc/current/components/uid.html

A uuid is preferable in the sense that when you create your object in your code, you won't have to wait for the entity to be saved in the database to use your id.
Typically in the case where you would have to dispatch an event with the id of the user (for example to send an email during creation), this avoids having to recover the entity in db after its insertion

Also, it is much more complicated to guess a uuid than an id. In a url, to edit a user for example, if the roles are not well defined, a user could edit another user if he knows his id (or by randomly writing 1, 2, etc.) while he is very unlikely to find a uuid